sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -0,0 +1,257 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint:disable=protected-access
+from typing import Optional
+from msrest.serialization import UTC
+import datetime as dt
+import uuid
+import json
+
+from ._generated.models._models import StorageBlobCreatedEventData, \
+    EventGridEvent as InternalEventGridEvent, \
+    CloudEvent as InternalCloudEvent
+
+class CloudEvent(InternalCloudEvent):   #pylint:disable=too-many-instance-attributes
+    """Properties of an event published to an Event Grid topic using the CloudEvent 1.0 Schema.
+
+    All required parameters must be populated in order to send to Azure.
+
+    :param source: Required. Identifies the context in which an event happened. The combination of id and source must be 
+        unique for each distinct event. If publishing to a domain topic, source must be the domain name.
+    :type source: str
+    :param data: Event data specific to the event type.
+    :type data: object
+    :param type: Required. Type of event related to the originating occurrence.
+    :type type: str
+    :param time: The time (in UTC) the event was generated, in RFC3339 format.
```

Comment:
we typically do a time_utc naming pattern nowdays, assuming this passes Laurent's scream-test

Reply:
is it okay if we stray from CloudEvent convention, and call it time_utc? Even though time field in CloudEvent is "time", not "time_utc"

Reply:
That's a very good point.  This isn't 'our api surface'; we're presenting a shared contract.  Hrm.  I think I have to agree with you.